internal/pkg/cli/pipeline_show.go

```diff
@@ -181,7 +184,7 @@ func (o *showPipelineOpts) askPipelineName() error {
 }
 
 func (o *showPipelineOpts) retrieveAllPipelines() ([]string, error) {
```

Comment:
Just a nit but maybe defining this function adds little value.

Reply:
Is this a blocker for you?